### ISSUE TITLE
Error: line contains NUL

### ISSUE THREAD

**Issue by leepand** [citrusvanilla/tinyflux#14]
hi, when i run the following codes, got errors

from tinyflux import TinyFlux,TagQuery,FieldQuery
db = TinyFlux('tm_db/features_store.csv')

---------------------------------------------------------------------------
Error                                     Traceback (most recent call last)
File <timed exec>:2, in <module>

File ~/miniconda3/lib/python3.8/site-packages/tinyflux/database.py:181, in TinyFlux.__init__(self, auto_index, *args, **kwargs)
    179 # Reindex if auto_index is True.
    180 if self._auto_index and not self._storage._initially_empty:
--> 181     self.reindex()

File ~/miniconda3/lib/python3.8/site-packages/tinyflux/database.py:754, in TinyFlux.reindex(self)
    751     return
    753 # Build the index.
--> 754 self._index.build(
    755     self._storage._deserialize_storage_item(i) for i in self._storage
    756 )
    758 return

File ~/miniconda3/lib/python3.8/site-packages/tinyflux/index.py:185, in Index.build(self, points)
    182 # A buffer for the new timestamps and their storage positions.
    183 timestamp_buffer: List[Tuple[float, int]] = []
--> 185 for idx, point in enumerate(points):
    186     self._num_items += 1
    187     self._insert_measurements(idx, point.measurement)

File ~/miniconda3/lib/python3.8/site-packages/tinyflux/database.py:754, in <genexpr>(.0)
    751     return
    753 # Build the index.
--> 754 self._index.build(
    755     self._storage._deserialize_storage_item(i) for i in self._storage
    756 )
    758 return

Error: line contains NUL

**Comment by citrusvanilla** [comment]
Looks like you might have a bad CSV file: [Python CSV error: line contains NULL byte](https://stackoverflow.com/questions/4166070/python-csv-error-line-contains-null-byte)

What does your CSV file look like?

**Comment by leepand** [comment]
<img width="848" alt="image" src="https://user-images.githubusercontent.com/26640201/184281749-7a82367a-a05c-4d42-92d7-c90678a9ea7e.png">

like this

**Comment by citrusvanilla** [comment]
Are you able to send the actual CSV?

**Comment by leepand** [comment]
@citrusvanilla
[features_store.csv.zip](https://github.com/citrusvanilla/tinyflux/files/9316918/features_store.csv.zip)

**Comment by citrusvanilla** [comment]
@leepand 

Okay I played around with the file, there is indeed a NUL byte in in.  This is likely to have occurred if you opened "features_store.csv" in Excel and saved it as a CSV from there.  I checked to see if the CSV was encoded with UTF-18 perhaps, but that wasn't the case.  In any event, the CSV should be encoded as UTF-8 as that is the default encoding of the CSV module in Python, so if you make edits to the DB in other applications, make sure it is still saved as UTF-8.

I removed the null bytes from your file and it appears to be properly encoded now.

https://stackoverflow.com/a/59912357

[features_store_utf8.csv.zip](https://github.com/citrusvanilla/tinyflux/files/9339056/features_store_utf8.csv.zip)
